R/parameters.R

```diff
@@ -405,7 +407,9 @@ get_parameters <- function(overrides = list()) {
     severe_incidence_rendering_min_ages = numeric(0),
     severe_incidence_rendering_max_ages = numeric(0),
     # misc
+    custom_demography = FALSE,
```

Comment:
I think we should reference the average_age variable here (line #91). If `custom_demography = FALSE` then the model uses average_age = 7663 which is the exponential distribution, right? So does `custom_demography = TRUE` override the average_age argument?